src/server.js

```diff
@@ -27,3 +34,54 @@ export function flushToHTML(options = {}) {
     return html
   }, '')
 }
+
+export function wrapWithProvider(element, registry) {
+  return React.createElement(
+    StylesheetRegistryContext.Provider,
+    { value: registry },
+    element
+  )
+}
+
+export function callDomServerMethod(reactDomServer, method, element) {
+  const registry = new StylesheetRegistry()
+  const wrapped = wrapWithProvider(element, registry)
+  const result = reactDomServer[method](wrapped)
+  return { result, registry }
+}
+
+export function renderToString(server, element) {
+  return callDomServerMethod(server, 'renderToString', element)
+}
+
+export function renderToStaticMarkup(server, element) {
+  return callDomServerMethod(server, 'renderToStaticMarkup', element)
+}
+
+export function renderToNodeStream(server, element) {
+  return callDomServerMethod(server, 'renderToNodeStream', element)
+}
+
+export function renderToStaticNodeStream(server, element) {
+  return callDomServerMethod(server, 'renderToStaticNodeStream', element)
+}
+
+export function wrapServer(server) {
+  return {
+    renderToString(element) {
+      return renderToString(server, element)
+    },
+
+    renderToStaticMarkup(element) {
+      return renderToStaticMarkup(server, element)
+    },
+
+    renderToNodeStream(element) {
+      renderToNodeStream(server, element)
+    },
+
+    renderToStaticNodeStream(element) {
+      renderToStaticNodeStream(server, element)
+    }
```

Comment:
Not sure if I should be exporting all these helper methods. I'd like feedback on which style you guys like before adding tests/documentation.